app/Services/Model/Imp/ScheduleService.php

```diff
@@ -403,4 +424,68 @@ public function publishAll(string $source, int $summit_id, array $payload, ?Filt
 
         return $schedule;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function addReview(Summit $summit, string $source, int $track_id, array $payload): SummitProposedSchedule
+    {
+        return $this->tx_service->transaction(function () use ($summit, $source, $track_id, $payload) {
+
+            $former_lock = $this->schedule_lock_repository->getBySummitAndTrackId($summit->getId(), $track_id);
+
+            if (!is_null($former_lock)) {
```

Comment:
is there is a former lock , then throw validation exception , not email, email is only for successful creation

Reply:
done